src/App.tsx

```diff
@@ -15,48 +17,120 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  noSort = '',
+  sortByLength = 'leng',
+  sortByName = 'name',
+}
+
+interface StateInterface {
+  sortType: SortType;
+  isReversed: boolean;
+}
+
+const sortGoods = (
+  goods: string[], { sortType, isReversed }: StateInterface,
```

Comment:
https://github.com/mate-academy/react_list-of-goods-hooks/pull/1242#discussion_r1315700445
There aren't that many arguments here to worry about a possible error in the order of arguments.
In addition, you still retain the importance of order (the array must always be first), so if you want to be sure that the order of the arguments is not important, you should pass all the arguments as one object, not just a part.